### PR TITLE
Feature/add socials block

### PR DESCRIPTION
**Pull request by Pisarskiy**

### CODE REVIEW

File: src/components/socials/index.js [hunk 1]
```diff
@@ -2,20 +2,31 @@ import React from 'react'
...
+import onscroll from './onscroll'
 
-export default Socials
+class SocialsSmart extends React.Component {
+  componentDidMount() {
+    onscroll()
+  }
+
+  render() {
+    const className = `${this.props.sidebar ? 'socials__sidebar' : ''}`
```
Comment (Evoltic) [inline src/components/socials/index.js]: According to BEM. Double underscope is used only for child elements. For this context you should use a modificator: 
socials_sidebar

File: src/components/socials/onscroll.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export default (onscroll = () => {
+  const elements = document.getElementsByClassName('socials__sidebar')
```
Comment (Evoltic) [inline src/components/socials/onscroll.js]: Document doesnt exist in prerender process

File: src/components/socials/socials.less [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    left: -1rem;
+    align-items: center;
+    z-index: 1;
+    top: 10%;
+
+    @media @under-sm {
+      left: -1rem;
```
Comment (Evoltic) [inline src/components/socials/socials.less]: You already declared left: -1rem;

File: src/components/socials/socials.less [hunk 4]
```diff
@@ -1,4 +1,24 @@
...
+    }
+
+    &.fixed {
```
Comment (Evoltic) [inline src/components/socials/socials.less]: &--fixed

File: src/pages/zmest/page_zmest.less [hunk 5]
```diff
@@ -265,6 +265,7 @@
     background-color: #fff;
     padding-top: 100px;
     padding-bottom: 100px;
+    position: relative;
```
Comment (Evoltic) [inline src/pages/zmest/page_zmest.less]: Positioning is higher than styling. http://codeguide.co/#css-declaration-order

File: src/components/socials/socials.less [hunk 6]
```diff
@@ -1,4 +1,24 @@
+@import '~src/common.blocks/imports/media.less';
+
 .socials {
+  &__sidebar {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    left: -1rem;
```
Comment (Evoltic) [inline src/components/socials/socials.less]: http://codeguide.co/#css-declaration-order

File: src/components/socials/onscroll.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+export default (onscroll = () => {
```
Comment (Evoltic) [inline src/components/socials/onscroll.js]: Overall. For this case you can use "position: sticky;" and avoid js at all

File: src/components/socials/socials.less (hunk 6 above)
Comment (Pisarskiy) [reply src/components/socials/socials.less]: fixed

File: src/pages/zmest/page_zmest.less (hunk 5 above)
Comment (Pisarskiy) [reply src/pages/zmest/page_zmest.less]: fixed

File: src/components/socials/socials.less (hunk 4 above)
Comment (Pisarskiy) [reply src/components/socials/socials.less]: fixed

File: src/components/socials/socials.less (hunk 3 above)
Comment (Pisarskiy) [reply src/components/socials/socials.less]: fixed

File: src/components/socials/onscroll.js (hunk 2 above)
Comment (Pisarskiy) [reply src/components/socials/onscroll.js]: fixed

File: src/components/socials/index.js (hunk 1 above)
Comment (Pisarskiy) [reply src/components/socials/index.js]: fixed

**Review by Evoltic — APPROVED** [review]

**Pull request merged**